src/libraries/Common/src/Interop/Unix/System.Native/Interop.ReadStdinUnbuffered.cs

```diff
@@ -13,7 +13,7 @@ internal static partial class Sys
         internal static extern unsafe int ReadStdin(byte* buffer, int bufferSize);
 
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_InitializeConsoleBeforeRead")]
-        internal static extern unsafe void InitializeConsoleBeforeRead(byte minChars = 1, byte decisecondsTimeout = 0);
+        internal static extern unsafe void InitializeConsoleBeforeRead(byte minChars = 1, byte decisecondsTimeout = 0, bool disableCrLfConversions = false);
```

Comment:
Maybe make `disableCrLfConversions` the first argument, and non-optional?

I wonder if there are cases where we want this to keep the previous set value (instead of using `false`)?

Reply:
> Maybe make disableCrLfConversions the first argument, and non-optional?

Out of curiosity, why would that be preferable? It seems like a less important parameter and also all others are optional too.

> I wonder if there are cases where we want this to keep the previous set value (instead of using false)?

No, all other use cases should preserve the current behaviour. It's just two, `ReadLine` and `TryGetCursorPosition`.

Reply:
> It seems like a less important parameter and also all others are optional too.

The other parameters indicate a read should block for at least 1 character. This is really the default behaviour of a `read`.
`disableCrLfConversions` controls what is returned. It depends on the case, and for me, this makes it less a default.

Reply:
addressed